pkg/config/setup/config.go

```diff
@@ -581,6 +581,15 @@ func InitConfig(config pkgconfigmodel.Setup) {
 	config.BindEnvAndSetDefault("cluster_agent.language_detection.cleanup.language_ttl", "30m")
 	// language annotation cleanup period
 	config.BindEnvAndSetDefault("cluster_agent.language_detection.cleanup.period", "10m")
+
+	config.BindEnvAndSetDefault("cluster_agent.appsec.injector.enabled", false)
```

Comment:
We’ll need to add this to the config templates to show it’s supported. But I think we should wait until it’s easier for users to enable the feature (helm/operator support for installing the processor, etc.).